auth/src/main/res/layout/welcome_back_password_prompt_layout.xml

```diff
@@ -1,63 +1,78 @@
+                android:text="@string/welcome_back_email_header"/>
 
-        </android.support.design.widget.TextInputLayout>
+            <TextView
+                style="@style/FirebaseUI.Text.BodyText"
+                android:id="@+id/welcome_back_password_body"/>
 
-    </RelativeLayout>
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
 
-    <LinearLayout
-        android:layout_height="wrap_content"
-        android:layout_width="match_parent"
-        android:paddingTop="16dp">
+                <android.support.design.widget.TextInputLayout
+                    android:id="@+id/password_layout"
+                    xmlns:app="http://schemas.android.com/apk/res-auto"
+                    style="@style/FirebaseUI.Text.TextInputLayout"
+                    android:paddingTop="16dp"
+                    app:hintTextAppearance="@style/FirebaseUI.Text.HintText"
+                    app:errorTextAppearance="@style/FirebaseUI.Text.ErrorText"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:passwordToggleEnabled="true">
 
-        <TextView
-            android:id="@+id/trouble_signing_in"
-            android:text="@string/trouble_signing_in"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_weight="1"
-            style="@style/FirebaseUI.Text.Link"/>
+                    <android.support.design.widget.TextInputEditText
+                        style="@style/FirebaseUI.EditText.PasswordField"
+                        android:id="@+id/password"
+                        android:hint="@string/password_hint"
+                        android:imeOptions="actionDone"/>
+
+                </android.support.design.widget.TextInputLayout>
+
+            </RelativeLayout>
+
+            <LinearLayout
+                android:layout_height="wrap_content"
+                android:layout_width="match_parent"
+                android:paddingTop="16dp">
+
+                <TextView
+                    android:id="@+id/trouble_signing_in"
+                    android:text="@string/trouble_signing_in"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    style="@style/FirebaseUI.Text.Link"/>
+
+            </LinearLayout>
+
+        </LinearLayout>
 
-        <Button
-            style="@style/FirebaseUI.FullWidthButtonSettleUp"
-            android:text="@string/sign_in_default"
-            android:id="@+id/button_done"/>
+    </ScrollView>
 
-    </LinearLayout>
+    <Button
+        style="@style/FirebaseUI.FullWidthButtonSettleUp"
+        android:text="@string/sign_in_default"
+        android:id="@+id/button_done"
+        android:background="@color/authui_colorAccent"/>
```

Comment:
Because it can't be orange when activity starts (on some screens). For example: Your  email adress is not filled, so the button should not be orange.